### PR TITLE
Updated swars

### PR DESCRIPTION
**Pull request by jeaaanius**

### CODE REVIEW

**Comment by forceporquillo** [comment]
pa fix muna nung conflicts.

File: project/swars/calendar.php [hunk 1]
```diff
@@ -205,12 +154,81 @@ function checkSlots($mysqli, $date){
...
+                        <form action="calendar.php" method="get">
+                            <button style="cursor:pointer;padding:8px;font-size:12px;border-radius:4px;background-color:#ff6666;color:white;border:none"name="del_button">CANCEL</button>
+                        </form>
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+        </div>
+    </section>
+
```
Comment (forceporquillo) [inline project/swars/calendar.php:231]: HULI KA HAHAHAHAHAHAHAHAHAH

File: project/swars/payment.php [hunk 2]
```diff
@@ -55,23 +55,23 @@
...
               <div class="col-50">
                 <label class="custom-field one" for="expyear">
-                <input type="text" id="expyear" name="expyear">
+                <input type="text" id="expyear" name="expyear" required value="2018">
                   <span class="placeholder"> Exp Year </span></label>
               </div>
               <div class="col-50">
                 <label class="custom-field one" for="cvv">
-                <input type="text" id="cvv" name="cvv">
+                <input type="text" id="cvv" name="cvv" required value="352">
```
Comment (forceporquillo) [inline project/swars/payment.php:74]: bat mo binalik yung value?

File: project/swars/server.php [hunk 3]
```diff
@@ -104,13 +104,6 @@
 }
 ?>
 <?php
-session_start();
-
-if(isset($_SESSION['logged_in'])) {
-    header('location: admin.php');
-}
-?>
-<?php
```
Comment (forceporquillo) [inline project/swars/server.php:113]: Thanks! ni-remove mo.

File: project/swars/calendar.php (hunk 1 above)
Comment (jeaaanius) [reply project/swars/calendar.php:231]: anmeron pu ahahhahhahahhahah

Comment (forceporquillo) [reply project/swars/calendar.php:231]: bat hard coded yan hahaha

**Review by forceporquillo — APPROVED** [review]

**Pull request merged**